### ISSUE TITLE
User Migration App

### ISSUE THREAD

**Issue by R7JANV1** [powerpaul17/nc_money#30]
Add the possibility to export and import data with the user Migration App.
https://apps.nextcloud.com/apps/user_migration
![grafik](https://user-images.githubusercontent.com/90777102/211209748-8e31ed06-5849-4c1f-84a4-e6db1e5425a3.png)

**Comment by powerpaul17** [comment]
Good idea, but since that should not be used very often, it is not currently a top priority..

**Comment by 1837456** [comment]
> ... it is not currently a top priority..

The export via user migration app or export in general?

**Comment by powerpaul17** [comment]
> > ... it is not currently a top priority..
> 
> The export via user migration app or export in general?

The export via user migration (also first time I heard of its existence..)

**Comment by 1837456** [comment]
> > > The export via user migration (also first time I heard of its existence..)

Me too.

By the way, I'm really happy about your app. It's great to have an app for accounting integrated into Nextcloud. Keep up the good work!
